container/README.md

```diff
@@ -0,0 +1,30 @@
+Building and using the container
+================================
+
+IMPORTANT: make sure your branch mentioned below is in GitHub! The local Git
+repo will be IGNORED by the Dockerfile.
+
+Assuming the container is build off the `master` branch of the package, do tis
+from the top directory of the package:
+```
+BRANCH=master
+cat container/Dockerfile.tmpl | sed 's/{{BRANCH}}/'${BRANCH}'/g' > container/Dockerfile
+docker build -t qserv/dax_data_generator:tools -f container/Dockerfile .
+docker push qserv/dax_data_generator:tools
+```
```

Comment:
I would suggest writing a shell script `container/build.sh` which would incorporate 4 lines above into something like this:
```
#!/bin/bash

set -e

CONTAINER_TOOLS_DIR="$0"
USAGE="
Usage:
    ${CONTAINER_TOOLS_DIR} <branch>

Where:
  <branch> - the name of the Git branch to build the container from. Note that if you
             have a local copy of the branch then commit and push your changes to
             GitHub before building the container.
             Note that the branch name will also be used to create a custom tag for the
             container. For example, branch 'tickets/DM-26409' will map to
             tag: 'qserv/dax_data_generator:tickets-DM-26207'. And branch 'master' will
             make tag: 'qserv/dax_data_generator:master'.
"
BRANCH="$1"
if [ -z "${BRANCH}" ]; then
    >&2 echo $USAGE
    exit 1
fi
TAG="qserv/dax_data_generator:$(echo $BRANCH | tr '/' '-')"
cd $CONTAINER_TOOLS_DIR
cat container/Dockerfile.tmpl | sed 's/{{BRANCH}}/'${BRANCH}'/g' > container/Dockerfile
docker build -t $TAG -f container/Dockerfile .
docker push $TAG
rm container/Dockerfile
```

Reply:
I added a build.sh file, it's a bit different than this one.